dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -360,13 +360,26 @@
 			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00100000>;
 			interrupts = <12 0>;
 			status = "disabled";
+			vref-mv = <3300>;
 			label = "ADC_1";
 			#io-channel-cells = <1>;
 			has-temp-channel;
 			has-vref-channel;
 			has-vbat-channel;
 		};
 
+		die_temp: dietemp {
+			compatible = "st,stm32-temp-cal";
+			ts-cal1-addr = <0x1FFF75A8>;
+			ts-cal2-addr = <0x1FFF75CA>;
+			ts-cal1-temp = <30>;
+			ts-cal2-temp = <130>;
+			ts-cal-vrefanalog = <3300>;
```

Comment:
please check the Datasheet: 
"TS_CAL1 and TS_CAL2 are raw data acquired at a temperature of 30 °C  and 130 °C   (± 5 °C),  VDDA = VREF+ = **3.0 V** (± 10 mV)"